app/server.go

```diff
@@ -510,6 +527,14 @@ func (s *Server) Start() error {
 	mlog.Info("Starting Server...")
 
 	var handler http.Handler = s.RootRouter
+
+	if *s.Config().LogSettings.EnableSentry {
+		sentryHandler := sentryhttp.New(sentryhttp.Options{
+			Repanic: true,
+		})
+		handler = sentryHandler.Handle(handler)
```

Comment:
This is only handling the panics, right? Do we need to create our own handler wrapper to store the 500 errors too?

Reply:
Yes, this is only for panics. Logging handled errors needs to be discussed a little more. (Privacy concerns?)